### PR TITLE
P2P-FE-BE-2 Update UI Offer/My Offer list to add SELL/BUY switch button and show BUY OFFFERS 

### PR DESCRIPTION
**Pull request by duongquang18it3**
# Ticket
https://github.com/mezonai/mmn-tx-explorer/issues/757

# General
- [ ] No unnecessary files, debug logs, or commented code included.
- [ ] PR has at least 1 peer approval before senior review.
- [ ] I fully understand the code I submitted.
- [ ] No hardcoded configuration or secrets.
- [ ] **Environment Variables** changes? (Leave unchecked if N/A)

# Design & Solution Clarity
- [ ] A design diagram (sequence / flow) is attached or linked.
- [ ] The implementation matches the design.
- [ ] All major flows and decision points are represented in the diagram.
- [ ] Error handling and edge cases are clearly defined in the design.

Design link (diagram / doc):
`<paste link here>`

# Testing
- [ ] Unit tests cover main logic and important edge cases.

> Note: Complex logic must not rely on manual testing only.

# UI evidence (For frontend only)
`<paste screenshots here>`

# Peer Review Checklist
- [ ] I understand the overall solution and flow.
- [ ] The code follows the provided design diagram.
- [ ] No obvious logic or security issues found.
- [ ] Tests are sufficient for the scope of this change.

### CODE REVIEW

File: dong-service/repository/p2p_order_repository.go [hunk 1]
```diff
@@ -23,8 +23,9 @@ func NewOrderRepository(db *sql.DB, dongSchema string) *OrderRepository {
 func (r *OrderRepository) CreateOrder(ctx context.Context, order *models.Order, tx *sql.Tx) error {
 	query := fmt.Sprintf(`
 			INSERT INTO %s.p2p_orders (
-				offer_id, buyer_wallet_address, buyer_user_id, order_amount, payable_amount, status, transfer_code, expires_at, created_at, updated_at
-			) VALUES ($1,$2,$3,$4,$5,$6,$7,$8,NOW(),NOW())
+				offer_id, buyer_wallet_address, buyer_user_id, order_amount, payable_amount, status, transfer_code, expires_at, created_at, updated_at,
+				offer_type, bank_info, seller_wallet_address, seller_user_id
```
Comment (vietdangquoc) [inline dong-service/repository/p2p_order_repository.go]: remove offer_type

File: dong-service/repository/p2p_order_repository.go [hunk 2]
```diff
@@ -119,7 +124,11 @@ func (r *OrderRepository) CancelExpiredOrders(ctx context.Context, cutoff time.T
 }
 
 func (r *OrderRepository) ListOrdersByOffer(ctx context.Context, offerID int64, pagination map[string]any) ([]models.Order, error) {
-	base := fmt.Sprintf("SELECT order_id, offer_id, buyer_wallet_address, buyer_user_id, order_amount, payable_amount, transaction_hash, status, transfer_code, expires_at, created_at, updated_at FROM %s.p2p_orders WHERE offer_id = $1", r.dongSchema)
+	base := fmt.Sprintf(`
+		SELECT order_id, offer_id, buyer_wallet_address, buyer_user_id, order_amount, payable_amount, transaction_hash, status, transfer_code, expires_at, created_at, updated_at,
+		       offer_type, bank_info, seller_wallet_address, seller_user_id
```
Comment (vietdangquoc) [inline dong-service/repository/p2p_order_repository.go]: remove offer type

File: dong-service/repository/p2p_order_repository.go [hunk 3]
```diff
@@ -119,7 +122,13 @@ func (r *OrderRepository) CancelExpiredOrders(ctx context.Context, cutoff time.T
 }
 
 func (r *OrderRepository) ListOrdersByOffer(ctx context.Context, offerID int64, pagination map[string]any) ([]models.Order, error) {
-	base := fmt.Sprintf("SELECT order_id, offer_id, order_creator_wallet_address, order_creator_user_id, order_amount, payable_amount, transaction_hash, status, transfer_code, expires_at, created_at, updated_at FROM %s.p2p_orders WHERE offer_id = $1 AND status IN ('PENDING', 'OPEN', 'CONFIRMED')", r.dongSchema)
+	base := fmt.Sprintf(`
+		SELECT o.order_id, o.offer_id, o.order_creator_wallet_address, o.order_creator_user_id, o.order_amount, o.payable_amount, 
+		       o.transaction_hash, o.status, o.transfer_code, o.expires_at, o.created_at, o.updated_at,
+		       of.offer_creator_wallet_address, of.offer_creator_user_id
```
[the remark this answers is not in the document]
Comment (duongquang18it3) [reply dong-service/repository/p2p_order_repository.go]: Filter theo status em nghĩ không cần vì đang cần show các order của một offer, thì cũng cần user nắm tổng quan anh ạ

[the remark this answers is not in the document]
Comment (duongquang18it3) [reply dong-service/repository/p2p_order_repository.go]: Dạ em đã update INNER JOIN @vietdangquoc

**Review by hoangdohuy — APPROVED** [review]
LGTM
